CHANGELOG.md

```diff
@@ -2,7 +2,17 @@
 
 All notable changes to this project will be documented in this file. The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org).
 
-## [v8.2.0](https://github.com/puppetlabs/puppetlabs-postgresql/tree/v8.2.0) (2022-08-22)
+## [v8.2.1](https://github.com/puppetlabs/puppetlabs-postgresql/tree/v8.2.1) (2022-08-23)
```

Comment:
Here is something I dislike about including the date in a file: it gets out of date quickly. Today it's already 2022-08-24, which you also saw with the 8.2.0 release: the changelog is changed again.

Reply:
Yeah, I agree its not ideal. We aim to get our releases out on the day they have been prepped, but it is just not always possible. 👎

Reply:
We have the same thing in our modules. Someone from Europe creates a PR, later someone from the US reviews and the next day it's merged & released. Quite often things are messy and cleaning this up with automation is non-trivial :(